src/main/java/org/esgi/boissibook/features/readlist/domain/BookProgression.java

```diff
@@ -0,0 +1,55 @@
+package org.esgi.boissibook.features.readlist.domain;
+
+public class BookProgression {
+    private final String bookProgressionId;
+    private final String bookId;
+    private final String userId;
+    private final Visibility visibility;
+    private final ReadingStatus readingStatus;
+    private final int currentPage;
+    private final int note;
+    private final String comment;
+
+    public BookProgression(String bookProgressionId, String bookId, String userId, Visibility visibility, ReadingStatus readingStatus, int currentPage, int note, String comment) {
+        this.bookProgressionId = bookProgressionId;
+        this.bookId = bookId;
+        this.userId = userId;
+        this.visibility = visibility;
+        this.readingStatus = readingStatus;
+        this.currentPage = currentPage;
+        this.note = note;
+        this.comment = comment;
+    }
```

Comment:
il me manque les required not null je les metterais quand j'irai plus in deep dans la logique